### DISCUSSION TITLE
Renovate bot doesn't seem to run on a regular schedule

### DISCUSSION THREAD

**Discussion by rcdailey** [renovatebot/renovate#29219] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

For my repository, I noticed that Renovate didn't run for 8 days. Once I pushed a change to my `master` branch, it started running again. The renovate dashboard shows that the last run was 8 days ago.

I assumed up to this point that the renovate bot had a default, regular schedule in which it runs, even if there is no repo activity. This is apparently a bad assumption. What is the default schedule configuration, if any? I wasn't able to find any docs on this, even on the scheduling page here:

https://docs.renovatebot.com/key-concepts/scheduling/

I expected the default behavior section to explain this stuff but it doesn't seem to.

Is `schedule` required in the JSON configuration to get *any* regular scans?

**Comment by rarkins** [comment]
Repos should normally see a job every 3-4 hours in the hosted app, unless they have been marked as stale/inactive or disabled.

**Reply by rcdailey** [reply]
That's definitely not what I was seeing. See screenshot below. I'll also provide a link to my repo, it's public.

![image](https://github.com/renovatebot/renovate/assets/1768054/ac65e184-3866-470d-9a0a-d79eb01807d0)

https://github.com/rcdailey/nezuko

Do you think there's a bug or a configuration issue on my end? I'm not sure how to diagnose.

**Reply by rarkins** [reply]
It looks like no jobs being scheduled. May be a sync issue on the backend.

**Reply by rarkins** [reply]
Our backend team will look into it

**Reply by nabeelsaabna** [reply]
From the last run:
```
{
 "repository":"rcdailey/nezuko",
 "msg":"Repository result: done, status: onboarded, enabled: true, onboarded: true",
 "time":"2024-05-23T13:58:09.039Z"
 }
```

`onboarded` means you didn't merge any Renvate update PRs yet and therefor you repository is schedueled to for a `daily` jobs by default

**Reply by rcdailey** [reply]
I already merged the onboard PR. And I wasn't even seeing daily runs, see my screenshot.

**Reply by nabeelsaabna** [reply]
history for the last few days:
```
2024-05-29 00:10:01  daily
2024-05-28 00:10:02  daily
2024-05-27 00:10:01  daily
2024-05-26 19:23:56  requested
2024-05-26 00:10:00  daily
2024-05-25 00:10:04  daily
2024-05-24 00:10:01  daily
```

**Reply by rcdailey** [reply]
@nabeelsaabna Please focus on the dates I provided in my screenshot, if possible. That was the time there was about an 8-day gap with no scans.

EDIT: Looks like there's no dates in the screenshot, I thought there were. Upon going back to my dashboard, unfortunately, it seems like those older scans are no longer available. I'm happy things are working pretty good now but I was really interested in RCA for the 8 day gap from before.

Unless you guys have access to older logs than I do on the frontend, it might be too late to diagnose it :(

**Reply by rarkins** [reply]
Unfortunately we do not have the time to provide RCAs for a free service. Thanks @nabeelsaabna for diligently following up

**Reply by rcdailey** [reply]
I wasn't asking for anything free; just trying to help you isolate a potential issue in your own product. This wouldn't really be for my benefit, as things seem to be working fine now. Either way, I appreciate your help and leave it up to you to determine the priority of this.
